Backpack/Backpack.h

```diff
@@ -28,6 +28,8 @@
 
     #import "Badge.h"
     #import "Card.h"
+    #import "DividedCard.h"
```

Comment:
These shouldn't be imported here, only one import per subspec here. These should be in `Card.h` and arguable `CardDivider` shouldn't be exported at all